website/integrations/services/index.mdx

```diff
@@ -10,7 +10,8 @@ import SupportBadge from "@site/src/components/SupportBadge";
 
 Below is a list of all applications that are known to work with authentik.
 
-:::info{title="Support Levels"}
+:::info
+
```

Comment:
@GirlBossRush I'd like to remove the edits for this file from this PR please, and let's chat about the changes. The big section about the the diff types of badges is too overwhelming, and I don't want to remove the section about Sources just yet, because they used to be here under Integrations, and so I want to link to them. I also want to look into the use of the cards for the Integration categories... I didn't even know that got changed.

Reply:
Reverted to include the title.